src/components/Form/Form.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+	Form,
+	StyledDiv,
+	StyledFieldset,
+	StyledInput,
+	StyledLegend,
+} from "./Form.Styled.ts";
+import { useState } from "react";
+import NavLinkButton from "../Button/NavLinkButton.tsx";
+
+function LoginForm() {
+	const [password, setPassword] = useState("");
+	const [name, setName] = useState("");
+
+	return (
+		<Form
+			onSubmit={(e) => {
+				e.preventDefault();
+			}}
+		>
+			<StyledFieldset>
+				<StyledLegend>Zaloguj się</StyledLegend>
+				<StyledDiv>
+					<StyledInput
+						type="text"
+						placeholder="Email"
+						value={name}
+						onChange={(e) => setName(e.target.value)}
+						required
+					/>
```

Comment:
Brakuje w tych inputach labeli lub oznaczeń że są to pola required, gwiazdka np.